packages/algoliasearch-helper/src/algoliasearch.helper.js

```diff
@@ -1509,6 +1511,17 @@ AlgoliaSearchHelper.prototype._change = function (event) {
   }
 };
 
+AlgoliaSearchHelper.prototype._recommendChange = function (event) {
```

Comment:
as we're not sure of the public api, I wonder if it should be `_recommend.change` and then we don't really need to be sparse with the number of methods

Reply:
I'm not sure I get this, could you elaborate? What would we gain in having this?

Reply:
thinking about it again, not sure if it's changing much, as we don't really need to document/type these "hidden" methods